test/rest.test.js

```diff
@@ -1398,23 +1399,22 @@ describe('strong-remoting-rest', function() {
         var method = givenSharedStaticMethod(
           function bar(cb) {
             var stringA = 'foo\xC1\xE1\u0102\u03A9asd><=$~!@#$%^&*()-_=+/.,;\'"[]{}?';
-            cb(null, { a: stringA });
+            cb(null, {a: stringA});
           },
           {
             returns: {
               arg: 'data', type: 'object', root: true,
-              xml: { wrapperElement: false },
+              xml: {wrapperElement: false},
             },
-            http: { path: '/' },
+            http: {path: '/'},
           }
         );
         request(app).get(method.classUrl)
           .set('Accept', 'application/xml')
           .set('Content-Type', 'application/json')
           .send()
-          .expect('Content-Type', /xml/)
+          .expect('Content-Type', /xml.*charset=utf-8/)
           .expect(200, function(err, res) {
-            expect(res).to.be.utf8;
```

Comment:
I had a WTF moment here. I think the intention was to assert that the response is in utf-8 encoding, which I am doing few lines above when checking the value of `Content-Type` header.